assets/components/displayName/displayName.jsx

```diff
@@ -4,28 +4,27 @@
 
 import React from 'react';
 import { connect } from 'react-redux';
-
 import { SvgUser } from 'components/svg/svg';
 
-
 // ---- Types ----- //
 
 type PropTypes = {
   name: string,
+  isSignedIn: boolean,
 };
 
 
 // ----- Component ----- //
 
 function DisplayName(props: PropTypes) {
-
-  return (
-    <div className="component-display-name">
-      <SvgUser />
-      <span className="component-display-name__name">{props.name}</span>
-    </div>
-  );
-
+  if (props.isSignedIn) {
+    return (
+      <div className="component-display-name">
+        <SvgUser />
+        <span className="component-display-name__name">{props.name}</span>
+      </div>
+    );
+  }
```

Comment:
I would add return null for the non-signed in case. In order to reduce the number of changes, I would structure the code in the following case, but this is totally a personal taste, feel free to do it or not:
```
if (!props.isSignedIn) {
    return null;
}
return ( 
    //template here
);
```